src/languages/python.ts

```diff
@@ -79,7 +82,10 @@ const nodeMatchers: Partial<Record<ScopeType, NodeMatcherAlternative>> = {
     ),
     patternMatcher("return_statement.~return!")
   ),
-  argumentOrParameter: argumentMatcher("parameters", "argument_list"),
+  argumentOrParameter: cascadingMatcher(
```

Comment:
Heyo! Cool! 

[Here's the link to the test case recorder](https://www.cursorless.org/docs/contributing/test-case-recorder/). `cursorless record` doesn't recognize well for me, just a heads up. 

A few good tests to record would be: 
- `"chuck arg"` with single argument in list
- `"chuck arg"` with multiple arguments in list, both generator expressions
- `"chuck arg"` with multiple arguments in list, one generator expression one other

I'm not super familiar with python but I assume that there are situations where we can have 2 generator expressions as arguments. If not, disregard that case.

Reply:
Is the idea behind these kinds of tests to record correct behavior and then running the tests will verify the behavior hasn't changed?

Reply:
I recorded and pushed a test case for 'take' and 'chuck' for each of the examples you gave. Any others I should add?

Reply:
Great!

I think a `chuck every argue` would be great. It looks like the `( )` and "," are cleaned up in multi argument situations but it might be good to have a case where we remove every argument when there are two.

I'll defer to @pokey on whether to chuck or keep the `"take"` tests. From my understanding, they're harder to parse since we need to look at the cursor positions to understand the output, and that's tougher than just looking and seeing that the argument has been deleted. If there isn't something in the contrib page, we should add it as a note for contributions.

Reply:
Our policy is as follows:

We like to have "change" tests for everything. It is not the end of the world to use "take" instead of "change", but we do slightly prefer "change" to "take" for the reasons @Will-Sommers mentions above. Generally not worth re-recording tho imo

For scope types where removal is interesting (eg ensuring comma gets removed), we like to also add "chuck" tests. I think this case does warrant "chuck" tests, as you've done

Reply:
And agreed a "chuck every" would be nice for good measure

Reply:
I've added one "chuck every" test with just a generator expression and one with the generator expression and another argument.

Reply:
Also I'll try to remember the preference for change instead of take for tests.